deploy/application/java/java-gradle.md

```diff
@@ -70,7 +72,7 @@ You can use the following properties:
     <tr>
       <td><span class="label label-default">Optional</span></td>
       <td>**build -&gt; type**</td>
-      <td>can be ``"maven"``, ``"gradle"`` or ``"ant"``</td>
+      <td>can be `"maven"`, `"gradle"` or `"ant"`</td>
```

Comment:
Since this is HTML, the backticks are rendered, I guess you can just drop them and maybe use the `<code>` instead.